curriculum/challenges/english/25-front-end-development/workshop-caesar-cipher/681a1d60e037127e7c8e0230.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 681a1d60e037127e7c8e0230
+title: Step 22
+challengeType: 20
+dashedName: step-22
+---
+
+# --description--
+
+You are going to use the parameter added in the previous step to determine if the function should encrypt the text passed to it (default behavior, `encrypt=True`), or if it should decrypt an encrypted message.
+
+Create an `if` statement that checks if `encrypt` is not truthy. Whitin the new `if` statement, set `shift` to `- shift`. This is necessary to enable the shift to take place in the opposite direction with respect to the encryption process.
+
+# --hints--
+
+You should set `shift` to `- shift` when `encrypt` is not truthy.
```

Comment:
I wasn't able to pass with this code:

```py
if encrypt is False:
    shift = -shift
```

Reply:
This will pass only with `if not encrypt`. I'll update the test.